tests/common/src/main/java/com/gentics/mesh/assertj/impl/JsonObjectAssert.java

```diff
@@ -80,8 +85,27 @@ public JsonObjectAssert hasNot(String path, String msg) {
 	public JsonObjectAssert has(String path, String value, String msg) {
 		try {
 			Object actualValue = getByPath(path);
-			String actualStringRep = String.valueOf(actualValue);
-			assertEquals("Value for property on path {" + path + "} did not match: " + msg, value, actualStringRep);
+			if (actualValue instanceof Collection<?>) {
```

Comment:
This enhancement allows assertions for value sets (order of the values is irrelevant)